server/src/lib.rs

```diff
@@ -279,26 +279,33 @@ impl InitData {
                                 trustify_module_graphql::endpoints::configure_graphiql(svc);
                             }),
                     );
-                    svc.app_data(web::Data::from(self.graph.clone()))
-                        .service(
-                            web::scope("/api")
-                                .wrap(new_auth(self.authenticator.clone()))
-                                .configure(|svc| {
-                                    trustify_module_importer::endpoints::configure(svc, db.clone());
-
-                                    trustify_module_fundamental::endpoints::configure(
-                                        svc,
-                                        db.clone(),
-                                        storage.clone(),
-                                    );
-                                }),
-                        )
-                        .configure(|svc| {
-                            // I think the UI must come last due to
-                            // its use of `resolve_not_found_to`
-                            #[cfg(feature = "ui")]
-                            trustify_module_ui::endpoints::configure(svc, &self.ui);
-                        });
+                    svc.app_data(web::Data::from(self.graph.clone())).service(
+                        web::scope("/api")
+                            .wrap(new_auth(self.authenticator.clone()))
+                            .configure(|svc| {
+                                trustify_module_importer::endpoints::configure(svc, db.clone());
+
+                                trustify_module_fundamental::endpoints::configure(
+                                    svc,
+                                    db.clone(),
+                                    storage.clone(),
+                                );
+                            }),
+                    );
+                    svc.app_data(web::Data::from(self.graph.clone())).service(
+                        web::scope("")
+                            .wrap(new_auth(self.authenticator.clone()))
+                            .configure(|svc| {
+                                trustify_module_importer::endpoints::configure(svc, db.clone());
+
+                                trustify_module_fundamental::endpoints::configure(
+                                    svc,
+                                    db.clone(),
+                                    storage.clone(),
+                                );
+                                trustify_module_ui::endpoints::configure(svc, &self.ui);
+                            }),
+                    );
```

Comment:
Can you explain what your intent is here? Configuring the same endpoints within two different scopes seems wrong.

Reply:
@jcrossley3, sorry I just simplified that. Please re-review.

Reply:
@jcrossley3, the intent should be clearer from the latest.
I believe the embedded UI should be clearly distinct from any API services the backend is providing.